frontend/src/components/Psychologist/Profile/EditProfile.jsx

```diff
@@ -55,24 +56,14 @@ const EditProfile = ({ psychologist, updatePsy, loading }) => {
           <h2>Informations pour l&lsquo;annuaire</h2>
           <Input
             label="Votre département"
-            type="select"
+            type="searchableSelect"
             field="departement"
             data-test-id="psy-departement-select"
             value={updatedPsychologist.departement}
             onChange={changePsychologist}
             options={DEPARTEMENTS.map(departement => ({ id: departement, label: departement }))}
             required
           />
-          <Input
-            label="Votre région"
-            type="select"
-            field="region"
-            data-test-id="psy-region-select"
-            value={updatedPsychologist.region}
-            onChange={changePsychologist}
-            options={REGIONS.map(region => ({ id: region, label: region }))}
-            required
-          />
```

Comment:
You removed region data, so we won't be able to change it anymore?
I would map each department with the correct region and update it accordingly (we can easily retrieve this map from geo api).
If we do that, we may want to check in db that every department/region are consistent.
What do you think?